strongbox-resources/strongbox-storage-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml

```diff
@@ -6,6 +6,11 @@
     <port>48080</port>
 
     <storages>
+        <storage id="nuget-common-storage">
+            <repositories>
+                <repository id="releases" policy="release" implementation="file-system" layout="Nuget Hierachlical" type="hosted" allows-force-deletion="true" checksum-headers-enabled="true"/>
```

Comment:
There is a typo here:

     <repository id="releases" policy="release" implementation="file-system" layout="Nuget Hierarchical" type="hosted" allows-force-deletion="true" checksum-headers-enabled="true"/>

(not `Hierachlical`).